odoo/addons/base/models/res_lang.py

```diff
@@ -88,8 +88,24 @@ def _register_hook(self):
         if not self.search_count([]):
             _logger.error("No language is active.")
 
-    @api.model
+    # TODO v14, remove me
```

Comment:
That's todo v15 (since the current master is the future v14).

Also nobody's going to check this anyway. If you want that done, put an assertion based on odoo.release.version_info.

Reply:
technically, master is `13.0alpha1`, I don't really care if we remove the method in 2 years or 2 weeks, just want to make sure nobody else is relying on it before removing it (warning should allow to detect that on the runbot)

Reply:
@mart-e it should be 14a1, @KangOl just forgot to bump the version of master after release.

See #38913